place_tracker/lib/place_map.dart

```diff
@@ -299,27 +307,6 @@ class _PlaceMapState extends State<PlaceMap> {
     });
   }
 
-  void _onPlaceChanged(Place value) {
```

Comment:
Removed as this logic is already implemented inside `place_details` and in the `_watchMapConfigurationChanges`